libs/core/src/components/sage-tooltip/stories/sage-tooltip.stories.js

```diff
@@ -0,0 +1,110 @@
+<div class="position-demo-grid">
+<div class="position-demo-grid-item"><sage-tooltip content="content 2" has-arrow=${args.hasArrow} placement="top-start" opened=${args.opened}>
+  <sage-button variant="accent">t</sage-button>
+</sage-tooltip>
+<sage-tooltip content="content 2" has-arrow=${args.hasArrow} placement="top" opened=${args.opened}>
+  <sage-button variant="accent">t</sage-button>
+</sage-tooltip>
+<sage-tooltip content="content 3" has-arrow=${args.hasArrow} placement="top-end" opened=${args.opened}>
+  <sage-button variant="accent">te</sage-button>
+</sage-tooltip>
+</div>
+<div class="position-demo-grid-item">
+<sage-tooltip content="content 1" has-arrow=${args.hasArrow} placement="left-start" opened=${args.opened}>
+  <sage-button variant="accent">ls</sage-button>
+</sage-tooltip>
+<sage-tooltip content="content 2" has-arrow=${args.hasArrow} placement="left" opened=${args.opened}>
+  <sage-button variant="accent">l</sage-button>
+</sage-tooltip>
+<sage-tooltip content="content 3" has-arrow=${args.hasArrow} placement="left-end" opened=${args.opened}>
+  <sage-button variant="accent">le</sage-button>
+</sage-tooltip>
+</div>
+<div class="position-demo-grid-item">
+<sage-tooltip content="content 1" has-arrow=${args.hasArrow} placement="bottom-start" opened=${args.opened}>
+  <sage-button variant="accent">bs</sage-button>
+</sage-tooltip>
+<sage-tooltip content="content 2" has-arrow=${args.hasArrow} placement="bottom" opened=${args.opened}>
+  <sage-button variant="accent">b</sage-button>
+</sage-tooltip>
+<sage-tooltip content="content 3" has-arrow=${args.hasArrow} placement="bottom-end" opened=${args.opened}>
+  <sage-button variant="accent">be</sage-button>
+</sage-tooltip>
+</div>
+<div class="position-demo-grid-item">
+<sage-tooltip content="content 1" has-arrow=${args.hasArrow} placement="right-start" opened=${args.opened}>
+  <sage-button variant="accent">rs</sage-button>
+</sage-tooltip>
+<sage-tooltip content="content 2" has-arrow=${args.hasArrow} placement="right" opened=${args.opened}>
+  <sage-button variant="accent">r</sage-button>
+</sage-tooltip>
+<sage-tooltip content="content 3" has-arrow=${args.hasArrow} placement="right-end" opened=${args.opened}>
+  <sage-button variant="accent">re</sage-button>
+</sage-tooltip>
+</div>
+</div>
+</div>`;
+
+export const Default = BaseTemplate.bind({});
+Default.args = {
+  content: "The tooltip content",
+  disabled: false,
+  placement: "right",
+  slot: "target text"
+};
+Default.parameters = { ...defaultParameters };
+
+export const Slotted = SlottedTemplate.bind({});
+Slotted.args = {
+  htmlContent: true,
+  placement: "bottom-start",
+};
+Slotted.parameters = { ...defaultParameters }
+
+
+export const Positioning = PositionTemplate.bind({});
+PositionTemplate.args = {
+  content: "Trigger"
+};
+PositionTemplate.parameters = { ...defaultParameters }
+
+export const NoArrow = BaseTemplate.bind({});
+NoArrow.args = {
+  content: "The tooltip content",
+  disabled: false,
```

Comment:
is `disabled` needed if false?

Reply:
Right now, yes because it has a `defaultValue` set. Ideally I don't want to have to set a `defaultValue` so I'll revisit this

Reply:
After removing the need for that prop, this comment is good to go